src/components/Button/PictureButton.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <v-container>
+    <v-file-input
+      label="File input"
+      variant="solo-filled"
+      prepend-icon="mdi-camera"
+      v-model="chosenFile"
+      accept=".jpeg, .jpg, .png"
+      @change="emitPicture"
+    ></v-file-input>
+  </v-container>
+</template>
+<script setup>
+import {ref} from 'vue'
+const chosenFile = ref(null)
+const props = defineProps({isDisabled: Boolean})
```

Comment:
Are we using isDisabled? If not, let's remove it

Reply:
done